src/runtime-rs/crates/runtimes/src/manager.rs

```diff
@@ -597,3 +601,18 @@ fn update_component_log_level(config: &TomlConfig) {
         updated_inner
     });
 }
+
+fn determine_jailed(config: &TomlConfig) -> Result<String> {
+    let Some(hypervisor_config) = config.hypervisor.get(&config.runtime.hypervisor_name) else {
+        todo!()
```

Comment:
There seems to be uncertainty how to handle this, however (assuming I'm guessing the intent of the previous line correctly) this block will only execute if `hypervisor_name` points to an unconfigured hypervisor, right?  If so then that would clearly be a major misconfiguration, warranting a return with an `Err`.  Or am I misunderstanding completely?

Reply:
correct -- AFAIU todo!() will raise an error. If not, we should definitely raise an error there.